### DISCUSSION TITLE
[vite] WARNING: You are using the default mode for `options.ts` which is `"civet"` error

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [DanielXMoore/Civet#1647] [Q&A] [answered]
`WARNING: You are using the default mode for `options.ts` which is `"civet"`. This mode does not support all TS features. If this is intentional, you should explicitly set `options.ts` to `"civet"`, or choose a different mode.`

Is this an option for the vite plugin? 

![image](https://github.com/user-attachments/assets/950d0439-e78e-4efb-8b3e-4239583ad818)

**Comment by edemaine** [comment, marked as answer]
Yes, it's `ts`, which you can see in the autocomplete.

If you're using Vite, which handles TypeScript itself, you can use `ts: "preserve"`.
